config/externalname.go

```diff
@@ -1086,6 +1086,9 @@ var TerraformPluginSDKExternalNameConfigs = map[string]config.ExternalName{
 	// No import
 	"aws_vpn_gateway_route_propagation": config.IdentifierFromProvider,
 
+	// verified access
+	"aws_verifiedaccess_trust_provider": config.IdentifierFromProvider,
```

Comment:
Since this is part of the ec2 package in the aws go sdk, it should go above `aws_vpc`. Please also include a comment with information from the terraform provider docs about how it is imported.